### DISCUSSION TITLE
Official API support chat now.

### DISCUSSION THREAD

**Discussion by JA1E0** [Zero6992/chatGPT-discord-bot#212] [Ideas]
We may not need to rely on the revChatGPT moudel anymore.
https://platform.openai.com/docs/guides/chat/introduction
<img width="1140" alt="image" src="https://user-images.githubusercontent.com/23023918/222376989-0cd36aaf-0426-41de-a684-7fb4ab23c812.png">

**Comment by Zero6992** [comment]
The new API has been integrated, but additional wrapping is required to support conversation with the native one

revChatGPT has already done this and I will keep using its wrapped API for now
